pymatgen/io/abinit/pseudos.py

```diff
@@ -843,13 +844,15 @@ def tm_header(filename, ppdesc):
         # 0   4.085   6.246    0   2.8786493        l,e99.0,e99.9,nproj,rcpsp
         # .00000000    .0000000000    .0000000000    .00000000   rms,ekb1,ekb2,epsatm
         projectors = {}
+        proj_info = []
+        idx = None
         for idx in range(2 * (lmax + 1)):
             line = lines[idx]
             if idx % 2 == 0:
                 proj_info = [
                     line,
                 ]
-            if idx % 2 == 1:
+            elif idx % 2 == 1:
```

Comment:
```suggestion
            elif idx % 2 == 1:
```

I am confused by this change as well. I don't think elif is correct here

Reply:
Thanks for catching this! 

This looks correct but unnecessary to me? (`idx % 2` could be either 0 or 1) Can you elaborate on your reason?

 I would remove the redundant check.

Reply:
Ah. Sorry. I also misunderstood something here. Yes, you are right.

Reply:
No worries at all. Thanks for asking and you still help me find a small improvement for this 😄